Sources/XCBuildSupport/XCBuildDelegate.swift

```diff
@@ -81,7 +81,7 @@ extension XCBuildDelegate: XCBuildOutputParserDelegate {
             }
         case .taskComplete(let info):
             queue.async {
-                self.buildSystem.delegate?.buildSystem(self.buildSystem, didStartCommand: BuildSystemCommand(name: "\(info.taskID)", description: info.result.rawValue))
+                self.buildSystem.delegate?.buildSystem(self.buildSystem, didFinishCommand: BuildSystemCommand(name: "\(info.taskID)", description: info.result.rawValue))
```

Comment:
@neonichu @abertelrud this sees correct on the surface, but not sure if there are some subtleties that required this less-intuitive usage

Reply:
I wonder if this is because of how SwiftPM's logging works? Since we do log in `didStartCommand`, maybe this was done to "fix" the log output when using XCBuild.